tracing-forest/src/printer/pretty.rs

```diff
@@ -94,9 +96,13 @@ impl Pretty {
         write!(writer, "{} ", shared.uuid)?;
 
         #[cfg(feature = "chrono")]
-        write!(writer, "{:<32} ", shared.timestamp.to_rfc3339())?;
+        write!(writer, "{:<36} ", shared.timestamp.to_rfc3339())?;
 
-        write!(writer, "{:<8} ", shared.level)
+        #[cfg(feature = "ansi")]
+        return write!(writer, "{} ", ColorLevel(shared.level));
```

Comment:
nit: instead of having manual right-padding in the `Display` impl, we should do padding here with `"{:<8} "` like below on L105

Reply:
Done, I had to do a little bit of fiddling with the `Display` implementation to get it to work properly with the color encoding characters.